config/config.go

```diff
@@ -48,6 +48,35 @@ const (
 
 var LoadBalancingStrategies = []string{LOAD_BALANCE_RR, LOAD_BALANCE_LC}
 
+// ServiceBrokerConfig configuration parameters
+type ServiceBrokerConfig struct {
+	ID               string
+	Name             string
+	Description      string
+	DisplayName      string
+	LongDescription  string
+	DocumentationURL string
+	SupportURL       string
+	ImageURL         string
+	ProviderName     string
+	Username         string `yaml:"username"`
```

Comment:
Move this to a top level config and make it required. Update StatusConfig to show being deprecated. If a StatusConfig is provided use that to lock down the /routes and top level auth for broker, otherwise use the new top level auth config for both /routes and broker.

Reply:
Moved to EndpointUsername and EndpointPassword in top level config.